main.go

```diff
@@ -267,6 +267,7 @@ func main() {
 
 		if catabalancerEnabled && nodeStatsDB != nil {
 			if cli.Tags["node"] == "media" { // don't announce load balancing availability for testing nodes
+				mist := clients.NewMistAPIClient(cli.MistUser, cli.MistPassword, cli.MistHost, cli.MistPort, catabalancer.StatsUpdateInterval-time.Second)
```

Comment:
Could you add a comment with the reasoning why it's `catabalancer.StatsUpdateInterval-time.Second`? It's not clear to me.